packages/cli/src/util/link/repo.ts

```diff
@@ -0,0 +1,269 @@
+      yes
+    );
+    client.config.currentTeam = org.type === 'team' ? org.id : undefined;
+
+    const remoteUrls = await getRemoteUrls(
+      join(rootPath, '.git/config'),
+      output
+    );
+    if (!remoteUrls) {
+      throw new Error('Could not determine Git remote URLs');
+    }
+    const remoteNames = Object.keys(remoteUrls);
+    let remoteName: string;
+    if (remoteNames.length === 1) {
+      remoteName = remoteNames[0];
+    } else {
+      // Prompt user to select which remote to use
+      const originIndex = remoteNames.indexOf('origin');
+      const answer = await client.prompt({
+        type: 'list',
+        name: 'value',
+        message: 'Which Git remote should be used?',
+        choices: remoteNames.map(name => {
+          return { name: name, value: name };
+        }),
+        default: originIndex === -1 ? 0 : originIndex,
+      });
+      remoteName = answer.value;
+    }
+    const repoUrl = remoteUrls[remoteName];
+    output.spinner(
+      `Fetching Projects for ${link(repoUrl)} under ${chalk.bold(org.slug)}…`
+    );
+    // TODO: Add pagination to fetch all Projects
+    const query = new URLSearchParams({ repoUrl, limit: '100' });
+    const projects: Project[] = await client.fetch(`/v2/projects?${query}`);
+    if (projects.length === 0) {
+      output.log(
+        `No Projects are linked to ${link(repoUrl)} under ${chalk.bold(
+          org.slug
+        )}.`
+      );
+      // TODO: run detection logic to find potential projects.
+      // then prompt user to select valid projects.
+      // then create new Projects
+    } else {
+      output.log(
+        `Found ${chalk.bold(projects.length)} ${pluralize(
+          'Project',
+          projects.length
+        )} linked to ${link(repoUrl)} under ${chalk.bold(org.slug)}:`
+      );
+    }
+
+    for (const project of projects) {
+      output.print(`  * ${chalk.cyan(`${org.slug}/${project.name}\n`)}`);
+    }
+
+    shouldLink =
+      yes ||
+      (await confirm(
+        client,
+        `Link to ${projects.length === 1 ? 'it' : 'them'}?`,
+        true
+      ));
+
+    if (!shouldLink) {
+      output.print(`Canceled. Repository not linked.\n`);
+      return;
+    }
+
+    repoConfig = {
+      orgId: org.id,
+      remoteName,
+      projects: projects.map(project => {
+        return {
+          id: project.id,
+          name: project.name,
+          directory: normalize(project.rootDirectory || ''),
+        };
+      }),
+    };
+    await outputJSON(repoConfigPath, repoConfig, { spaces: 2 });
+    // TODO: add `README.txt`
+
+    // TODO: add to `.gitignore`
+    let isGitIgnoreUpdated = false;
+
+    output.print(
+      prependEmoji(
+        `Linked to ${link(repoUrl)} under ${chalk.bold(
+          org.slug
+        )} (created ${VERCEL_DIR}${
+          isGitIgnoreUpdated ? ' and added it to .gitignore' : ''
+        })`,
+        emoji('link')
+      ) + '\n'
+    );
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏Until we implement this, can we log that the user should do it?

```suggestion
    );
    console.log(`Created ${repoConfigPath}. Ensure ${VERCEL_DIR} is added to your ".gitignore".`);
```

Reply:
I added the gitignore logic now.